1234/en.md

```diff
@@ -0,0 +1,61 @@
+# LOJ 1234 - Harmonic Number
+
+## Solution
+From the definition of H<sub>n</sub> we perceive: **H<sub>n+1</sub> = H<sub>n</sub> + (1/ n)**
+
+So we need not to calculate **H<sub>n</sub>** using the straighforward formula for every single test case. We go from non-decreasing order of test cases whilst keeping a variable for harmonic sum so far. This way not only we save time but memory as well, because there's no need for storing all the harmonic values beforehand.
+
+## Complexity
+- Time Complexity: O($T$). 
+- Memory Complexity: O($T$).
+
+## Code
+
+### C++
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+
+typedef pair <int, int> pii;
```

Comment:
See if you can drop this typedef and use `pair<int, int>` directly in your code.